### PR TITLE
Refine  cn document, test=document_preview

### PR DESCRIPTION
**Pull request by LielinJiang**
Refine cn document of Normal, Uniform, iou_similarity, affine_channel.

### CODE REVIEW

**Comment by LielinJiang** [comment]
image

image
image
image


image


image
image
image

File: doc/fluid/api_cn/layers_cn/iou_similarity_cn.rst [hunk 1]
```diff
@@ -13,12 +13,12 @@ iou_similarity
     IOU(A, B) = \frac{area(A\cap B)}{area(A)+area(B)-area(A\cap B)}
 
 参数：
-    - **x** (Variable,默认LoDTensor,float类型) - 框列表X是二维LoDTensor，shape为[N,4],存有N个框，每个框代表[xmin,ymin,xmax,ymax],X的shape为[N,4]。如果输入是图像特征图,[xmin,ymin]市框的左上角坐标，接近坐标轴的原点。[xmax,ymax]是框的右下角坐标。张量可以包含代表一批输入的LoD信息。该批的一个实例能容纳不同的项数
-    - **y** (Variable,张量，默认float类型的张量) - 框列表Y存有M个框，每个框代表[xmin,ymin,xmax,ymax],X的shape为[N,4]。如果输入是图像特征图,[xmin,ymin]市框的左上角坐标，接近坐标轴的原点。[xmax,ymax]是框的右下角坐标。张量可以包含代表一批输入的LoD信息。
+    - **x** (Variable) - 框列表X是二维LoDTensor，shape为[N,4]，存有N个框，每个框表示为[xmin,ymin,xmax,ymax]，X的shape为[N,4]。如果输入是图像特征图,[xmin,ymin]表示框的左上角坐标，接近坐标轴的原点。[xmax,ymax]表示框的右下角坐标。该张量包含批次输入的LoD信息。该批次输入的一个实例能容纳不同的项数。数据类型为float32或float64。
+    - **y** (Variable) - 框列表Y是二维张量，存有M个框，每个框表示为[xmin,ymin,xmax,ymax]，Y的shape为[M,4]。如果输入是图像特征图，[xmin,ymin]表示框的左上角坐标，接近坐标轴的原点。[xmax,ymax]表示框的右下角坐标。数据类型为float32或float64。
 
-返回：iou_similarity操作符的输出，shape为[N,M]的张量，代表一对iou分数
+返回：shape为[N,M]的LoDTensor，代表每一对iou分数，数据类型与x相同
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/iou_similarity_cn.rst]: shape -> 维度， 其余地方同

File: doc/fluid/api_cn/layers_cn/iou_similarity_cn.rst [hunk 2]
```diff
@@ -13,12 +13,12 @@ iou_similarity
     IOU(A, B) = \frac{area(A\cap B)}{area(A)+area(B)-area(A\cap B)}
 
 参数：
-    - **x** (Variable,默认LoDTensor,float类型) - 框列表X是二维LoDTensor，shape为[N,4],存有N个框，每个框代表[xmin,ymin,xmax,ymax],X的shape为[N,4]。如果输入是图像特征图,[xmin,ymin]市框的左上角坐标，接近坐标轴的原点。[xmax,ymax]是框的右下角坐标。张量可以包含代表一批输入的LoD信息。该批的一个实例能容纳不同的项数
-    - **y** (Variable,张量，默认float类型的张量) - 框列表Y存有M个框，每个框代表[xmin,ymin,xmax,ymax],X的shape为[N,4]。如果输入是图像特征图,[xmin,ymin]市框的左上角坐标，接近坐标轴的原点。[xmax,ymax]是框的右下角坐标。张量可以包含代表一批输入的LoD信息。
+    - **x** (Variable) - 框列表X是二维LoDTensor，shape为[N,4]，存有N个框，每个框表示为[xmin,ymin,xmax,ymax]，X的shape为[N,4]。如果输入是图像特征图,[xmin,ymin]表示框的左上角坐标，接近坐标轴的原点。[xmax,ymax]表示框的右下角坐标。该张量包含批次输入的LoD信息。该批次输入的一个实例能容纳不同的项数。数据类型为float32或float64。
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/iou_similarity_cn.rst]: [xmin,ymin,xmax,ymax]  空下格吧，最好用 :math:`` 括起来

File: doc/fluid/api_cn/layers_cn/iou_similarity_cn.rst (hunk 1 above)
Comment (LielinJiang) [reply doc/fluid/api_cn/layers_cn/iou_similarity_cn.rst]: done

File: doc/fluid/api_cn/layers_cn/iou_similarity_cn.rst (hunk 2 above)
Comment (LielinJiang) [reply doc/fluid/api_cn/layers_cn/iou_similarity_cn.rst]: done

**Review by heavengate — APPROVED** [review]

**Review by ceci3 — APPROVED** [review]

**Pull request merged**